packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.tsx

```diff
@@ -319,15 +324,56 @@ export const YppAuthorizationModal: FC<YppAuthorizationModalProps> = ({
     [fetchedChannelRequirements, ytRequirementsErrors]
   )
 
+  const verifyChannelRequirements = useCallback(async () => {
+    const channels = activeMembership?.channels
+
+    if (!channels?.length) {
+      navigate(absoluteRoutes.studio.newChannel())
+      return
+    }
+    setIsFetchingData(true)
+    const { data } = await refetchYppSyncedChannels().finally(() => {
+      setIsFetchingData(false)
+    })
+    if (data?.currentChannel) {
+      navigate(absoluteRoutes.studio.ypp())
+      return
+    }
+
+    if (data?.unsyncedChannels?.length) {
+      setSelectedChannelId(data?.unsyncedChannels[0].id)
+    }
+    if (data?.unsyncedChannels?.length && data?.unsyncedChannels.length > 0) {
```

Comment:
As far as I remember we should show `select-channel` step only if the member has more than one channel https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5/YouTube-Partner-Program?node-id=1637-121102&t=bi75D366Rqaa2LFi-4

Reply:
Nice catch! Do you think we should change the button text? Now we always display `Select channel`, even tho if there is only 1 channel we will go straight to google auth. Maybe smth more generic, like `Proceed`?

![image](https://user-images.githubusercontent.com/92513933/228771279-0fca4607-0dde-474c-9041-c361de4d0343.png)

Reply:
It's not really true, since now it's either Auth or select channel and we don't have information which one till user clicks the button

Reply:
We agreed together that we use `Continue` in this case